src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -3,6 +3,7 @@
 Release History
 ===============
 * webapp: az webapp ssh handles 'AZURE_CLI_DISABLE_CONNECTION_VERIFICATION' environment variable
+* appserviceplan: az appserviceplan create support for Linux FREE sku
```

Comment:
can you please fix the text like `[webapp up]: add free plan region check, and add 30 seconds wait on app-setting change`